extensions/blocks/podcast-player/templates/playlist-track.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Podcast Title template.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Podcast_Player;
+
+/**
+ * Template variables.
+ *
+ * @var array $attachment
+ * @var array $primary_colors
+ * @var array $secondary_colors
+ * @var bool  $is_active
+ */
+
+$track_title    = $attachment['title'];
```

Comment:
Can that title be empty as well, like the duration below?

Reply:
we default the title inside the helper to a `(no title)` string